src/logger/__tests__/MetricsContext.test.ts

```diff
@@ -385,8 +385,8 @@ test.each([
   [faker.random.words(3), faker.datatype.number(), Unit.Seconds, undefined],
   ['Max_Value', Number.MAX_SAFE_INTEGER, Unit.Milliseconds, undefined],
   ['-Max_Value', -Number.MAX_SAFE_INTEGER, 'Bytes/Second', undefined],
-  ['-Max_Value', -Number.MAX_SAFE_INTEGER, 'Bytes/Second', 1],
-  ['-Max_Value', -Number.MAX_SAFE_INTEGER, 'Bytes/Second', 60],
+  ['-Max_Value', faker.datatype.number(), 'Bytes/Second', 1],
+  ['-Max_Value', faker.datatype.number(), 'Bytes/Second', 60],
```

Comment:
Why did you change from `-Number.MAX_SAFE_INTEGER` to `faker.datatype.number()`?

Reply:
The last two lines of the test verify storage resolution and they are not to verify edge cases for the unit parameter (they are getting verified in the lines above). So I rectified my mistake by updating the unit value to some random value instead of an edge case.